### ISSUE TITLE
Move photo's to an album, searching for correct album looks like it's randiom.

### ISSUE THREAD

**Issue by MaikelvanGorkom** [Piwigo/Piwigo#2249]
I use an unsorted album to check images and after that I move selected pictures to the correct album. I see that the album which partial name i entered is shown, but i notice that the albums are in a (more or less) random fashion. So i see albums that should be alphabetically later, further up the list.

example
Car / Ferrari / F40 / Winter scenery
Trees / Oak / Winter
Car / Ferrari / F1 / Winter test

Would expect:
Car / Ferrari / F1 / Winter test
Car / Ferrari / F40 / Winter scenery
Trees / Oak / Winter

**Comment by plegall** [comment]
@LintyDev I guess it's in the album selector. Is the list of albums sorted by `global_rank`?

**Comment by LintyDev** [comment]
> @LintyDev I guess it's in the album selector. Is the list of albums sorted by `global_rank`?

Yes, by default the results are sorted by `global_rank` as shown on the screen below

![Capture d’écran 2024-10-25 à 11 25 20](https://github.com/user-attachments/assets/29b19a5b-ca96-4360-97a9-ab7951bb4dc8)

I tried to reproduce your environment and do some research and the results (in my case) look good. Can you make some screenshots so we can see.
![Capture d’écran 2024-10-25 à 11 37 40](https://github.com/user-attachments/assets/a22dc3f5-b247-4f7f-91b1-fba6cead7c55)
